### PR TITLE
fix(es/preset-env): move findLast and findLastIndex to finished proposals

### LINKED ISSUE #7001: Does not add imports to `findLast` and `findLastIndex` core-js polyfills when `{ env: { mode: 'usage' }}`
### Describe the bug

When using `{ env: { mode: 'usage' }}`, swc fails to add imports to `findLast` and `findLastIndex` core-js polyfills.

I've tried multiple options, to see if I am missing something but managed to get them there only by explicitly using the `include` option.

Thanks in advance for taking a look 👍.

### Input code

```typescript
const array = [1, 2, 3];
array.findLastIndex();
array.findLast();
```

### Config

```json
{
  "jsc": {
    "parser": {
      "syntax": "ecmascript",
      "jsx": false
    }
  },
  "module": {
    "type": "es6"
  },
  "isModule": true,
  "env": {
    "targets": "chrome >=63",
    "mode": "usage",
    "coreJs": "3.29"
  }
}
```

### Playground link

https://play.swc.rs/?version=1.3.37&code=H4sIAAAAAAAAA0vOzysuUUgsKkqsVLBViDbUUTDSUTCOteYCC%2BmlZeal%2BCQWl3jmpaRWaGiiCwNFAJfu0s1BAAAA&config=H4sIAAAAAAAAAy3M0QrCMAwF0F%2BRPBcfHAwc6AcIfkSo1zlZ15Kk4hj9d9vhW3LvSTZ6q6dho8SikDbpuhh%2FaSD4wOplSkausho9eVaU4ijER57RtK0JzWpPNZ%2F0%2Fm9MMhxh%2BeyIZYRpdf4lMeBwvfQd7W%2FacVYeUVcfBbemuuPpTKX8AGxXDAedAAAA

### Expected behavior

```js
import "core-js/modules/esnext.array.find-last-index.js";
import "core-js/modules/esnext.array.find-last.js";
const array = [
    1,
    2,
    3
];
array.findLastIndex();
array.findLast();
```

### Actual behavior

Actual output:

```js
const array = [
    1,
    2,
    3
];
array.findLastIndex();
array.findLast();
```

-> Polyfills are missing.

### Version

1.3.37

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by nissy-dev**
**Description:** Fix #7001

**BREAKING CHANGE:** No 

**Related issue (if exists):** #3970

### CODE REVIEW

**Comment by zloirock** [comment]
It seems it's time to completely update it by https://github.com/babel/babel-polyfills/blob/main/packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts

**Comment by nissy-dev** [comment]
Thanks! I will create annother PR for other updates

File: crates/swc_ecma_preset_env/tests/fixtures/corejs3/usage-finished-proposal/options.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
       "../../../../lib",
       {
         "useBuiltIns": "usage",
-        "corejs": "3.21",
```
Comment (kdy1) [inline crates/swc_ecma_preset_env/tests/fixtures/corejs3/usage-finished-proposal/options.json:7]: Please add another test and revert this file

Comment (nissy-dev) [reply crates/swc_ecma_preset_env/tests/fixtures/corejs3/usage-finished-proposal/options.json]: I reverted!

**Review by kdy1 — APPROVED** [review]
swc-bump:
 - swc_ecma_preset_env

**Pull request merged**